### PR TITLE
Add Fiverr

### PR DESCRIPTION
**Pull request by Nicos18**

### CODE REVIEW

**Review by mxxcon — CHANGES_REQUESTED** [review]
Their official logo
fiverr

**Review by mxxcon — APPROVED** [review]

File: _data/other.yml [hunk 1]
```diff
@@ -197,6 +197,13 @@ websites:
       software: Yes
       doc: https://www.findmyshift.com/help/what-is-two-factor-authentication
 
+    - name: Fiverr
+      url: https://www.fiverr.com/
+      img: fiverr.png
+      tfa: No
+      twitter: fiverr
```
Comment (Jawshy) [inline _data/other.yml]: As with https://github.com/2factorauth/twofactorauth/pull/2797#discussion_r142583002, should [`fiverr`](https://twitter.com/fiverr) be changed to [`FiverrSupport`](https://twitter.com/FiverrSupport)?

**Review by Jawshy — COMMENTED** [review]
`twitter`

File: _data/other.yml (hunk 1 above)
Comment (jamcat22) [reply _data/other.yml]: It seems that their support handle actively answers questions so that one would probably be best.

**Review by Jawshy — APPROVED** [review]

**Pull request merged**